packages/flutter_tools/templates/app/lib/main.dart.tmpl

```diff
@@ -56,7 +56,8 @@ class MyApp extends StatelessWidget {
         // or simply save your changes to "hot reload" in a Flutter IDE).
         // Notice that the counter didn't reset back to zero; the application
         // is not restarted.
-        primarySwatch: Colors.blue,
+        colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepPurple),
+        useMaterial3: true,
```

Comment:
why do we believe it's time to change the template but not time to change the default for `useMaterial3`?

Reply:
It's a big change that we're going to make incrementally.